framework_tvb/tvb/interfaces/rest/client/examples/__init__.py

```diff
@@ -0,0 +1,11 @@
+import sys
```

Comment:
Missing license, and the use of __init__.py is not ideal, but ok as an idea to accept input params